database/seeders/FeeStatementSeeder.php

```diff
@@ -22,23 +23,335 @@ public function run()
+            ],
+            [
+                'user_id' => '8',
+                'date' => '2020-06-07',
+                'document_number' => '128351',
+                'document_type' => 'Invoice', 
+                'type' => 'Debit',
+                'amount' => '100000.00',
+            ],
+            [
+                'user_id' => '8',
+                'date' => '2020-06-07',
+                'document_number' => '129052',
+                'document_type' => 'Receipt', 
+                'type' => 'Credit',
+                'amount' => '100000.00',
+            ],
+            //User 9
+            [
+                'user_id' => '9',
+                'date' => '2020-06-07',
+                'document_number' => '127363',
+                'document_type' => 'Invoice', 
+                'type' => 'Debit',
+                'amount' => '100000.00',
+            ],
+            [
+                'user_id' => '9',
+                'date' => '2020-06-07',
+                'document_number' => '127456',
+                'document_type' => 'Receipt', 
+                'type' => 'Credit',
+                'amount' => '100000.00',
+            ],
+            [
+                'user_id' => '9',
+                'date' => '2020-06-07',
+                'document_number' => '128351',
+                'document_type' => 'Invoice', 
+                'type' => 'Debit',
+                'amount' => '100000.00',
+            ],
+            [
+                'user_id' => '9',
+                'date' => '2020-06-07',
+                'document_number' => '129052',
+                'document_type' => 'Receipt', 
+                'type' => 'Credit',
+                'amount' => '100000.00',
+            ],
+            //User 10
+            [
+                'user_id' => '10',
+                'date' => '2020-06-07',
+                'document_number' => '127363',
+                'document_type' => 'Invoice', 
+                'type' => 'Debit',
+                'amount' => '100000.00',
+            ],
+            [
+                'user_id' => '10',
+                'date' => '2020-06-07',
+                'document_number' => '127456',
+                'document_type' => 'Receipt', 
+                'type' => 'Credit',
+                'amount' => '100000.00',
+            ],
+            [
+                'user_id' => '10',
+                'date' => '2020-06-07',
+                'document_number' => '128351',
+                'document_type' => 'Invoice', 
+                'type' => 'Debit',
+                'amount' => '100000.00',
+            ],
+            [
+                'user_id' => '10',
+                'date' => '2020-06-07',
+                'document_number' => '129052',
+                'document_type' => 'Receipt', 
+                'type' => 'Credit',
+                'amount' => '100000.00',
+            ],
 
         ];
 
         foreach ($fee_statements as $fee_statement) {
-            User::create([
```

Comment:
You cant do this. The users have already been created by the UserSeeder. You cant create them again. Just run the FeeStatement create only